applications/launchpad/backend/src/commands/host.rs

```diff
@@ -21,12 +21,28 @@
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
-use std::{convert::TryFrom, path::PathBuf, process::Command, time::Duration};
+use std::{
+    convert::TryFrom,
+    path::PathBuf,
+    process::{Command, Stdio},
+    time::Duration,
+};
 
+use bollard::Docker;
 use log::*;
 use tauri::{api::path::home_dir, AppHandle, Wry};
 
-use crate::commands::AppState;
+use crate::{commands::AppState, docker::DOCKER_INSTANCE};
+
+#[tauri::command]
+pub async fn check_docker(_app: AppHandle<Wry>) -> Result<String, String> {
+    let version = &DOCKER_INSTANCE.clone().version().await.map_err(|e| {
+        error!("Failed check docker version: {}", e);
+        format!("The docker version cannot be run")
+    })?;
+
+    Ok(format!("{:?}", version))
```

Comment:
Better to return JSON here. `Version` does implement `Serialize` so you can just return the object as is and you'll get JSON on the other end
```suggestion
    Ok(version)
```